Airgap installer changes in docs (#2301)
dapr/docs: daprdocs/content/en/reference/cli/dapr-init.md

| `--namespace`, `-n`  |                      | `dapr-system` | The Kubernetes namespace to install Dapr in                                          |
| `--runtime-version`  |                      | `latest`      | The version of the Dapr runtime to install, for example: `1.0.0`                     |
| `--slim`, `-s`       |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |

> | `--from-dir`         |                      |               | Path to local directory containing dapr artifacts to init in airgap environment      |

Reviewer: Should this not say "Dapr Installer Bundle" since this is the defined artifact package?

```suggestion
| `--from-dir`         |                      |               | Path to a local directory containing a downloaded "Dapr Installer Bundle" release which is used to `init` the airgap environment      |
```

dapr init -s
```

> In airgap environment, you can [download](https://github.com/dapr/installer-bundle/releases) bundled dapr artifacts and use it to install dapr instead of pulling from network.

Reviewer: ```suggestion
In an offline or airgap environment, you can [download a Dapr Installer Bundle](https://github.com/dapr/installer-bundle/releases) and use this to install Dapr instead of pulling images from the network.
```

```bash

> dapr init --from-dir <path-to-bundle-directory>

Reviewer: ```suggestion
dapr init --from-dir <path-to-installer-bundle-directory>
```

```

> Dapr can run slim self-hosted mode without Docker in airgap environment as well.

Reviewer: ```suggestion
Dapr can also run in slim self-hosted mode without Docker in an airgap environment.
```

```bash

> dapr init -s --from-dir <path-to-bundle-directory>

Reviewer: ```suggestion
dapr init -s --from-dir <path-to-installer-bundle-directory>
```
